homeassistant/components/zha/core/listeners.py

```diff
@@ -183,6 +183,7 @@ def zha_send_event(self, cluster, command, args):
             'zha_event',
             {
                 'unique_id': self._unique_id,
```

Comment:
From the PR description it sounds like we should remove (replace) the unique_id. Am I misunderstanding it?

Reply:
I worded the description poorly. They should both be there. I’ll amend the description.

Reply:
Updated